order/models.py

```diff
@@ -49,13 +50,22 @@ class Declaration(models.Model):
     is_paid = models.BooleanField(default=False)
     penalty_status = models.BooleanField(default=False)
     penalty = models.DecimalField(max_digits=4, decimal_places=2, default=0, null=True, blank=True)
-
     
-    def save(self, *args, **kwargs):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.__status = self.status.id if self.status else None
+        return 
```

Comment:
No need to return, end of function is return. And there is a ncie way to write line 56:
self.__status = self.status and self.status.id

Reply:
Forgot to delete return )